tests/node_test.rs

```diff
@@ -0,0 +1,55 @@
+use gclient::{EventProcessor, GearApi, Result};
+use gstd::Encode;
+use staking::io::InitStaking;
+
+#[cfg(debug_assertions)]
+const PATH: &str = "./target/wasm32-unknown-unknown/debug/staking.opt.wasm";
+
+#[cfg(not(debug_assertions))]
+const PATH: &str = "./target/wasm32-unknown-unknown/release/staking.opt.wasm";
+
+const USERS: &[u64] = &[1, 2, 3, 4, 5, 6, 7, 8];
+
+#[tokio::test]
+async fn init() -> Result<()> {
```

Comment:
I think one initialization scenario won't be enough. It's better to extend this test with some simple scenario with the `handle()` entrypoint.